openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -67,29 +81,40 @@ def send_credit_notifications(username, course_key):
     msg_alternative = MIMEMultipart('alternative')
     notification_msg.attach(msg_alternative)
     # render the credit notification templates
-    subject = _("Course Credit Eligibility")
+    subject = _('Course Credit Eligibility')
 
     # add alternative plain text message
     email_body_plain = render_to_string('credit_notifications/credit_eligibility_email.txt', context)
     msg_alternative.attach(MIMEText(email_body_plain, _subtype='plain'))
 
     # add alternative html message
-    email_body = cache.get('css-email-body')
-    if not email_body:
-        email_body = with_inline_css(
-            render_to_string("credit_notifications/credit_eligibility_email.html", context)
-        )
-        cache.set('css-email-body', email_body, settings.CREDIT_NOTIFICATION_CACHE_TIMEOUT)
-
+    email_body_content = cache.get('css-email-body')
+    if email_body_content is None:
+        html_file_path = file_path_finder('templates/credit_notifications/credit_eligibility_email.html')
+        if html_file_path:
+            with open(html_file_path, 'r') as cur_file:
+                cur_text = cur_file.read()
+                html_parser = HTMLParser.HTMLParser()
+                email_body_content = html_parser.unescape(with_inline_css(cur_text))
```

Comment:
It unescape just the html escaped characters by the `pynliner` e.g.,
```html
&lt;%! from django.utils.translation import ugettext as _ %&gt;
```
is changed back to:
```html
<%! from django.utils.translation import ugettext as _ %>
```
I have tested it locally with unicode html content and it works fine.